src/schemas/secrets.schema.json

```diff
@@ -38,7 +38,7 @@
     "EMBED_WHITELIST": {
       "type": "string",
       "description": "Comma-separated list of allowed hostnames for embeds. Supports `*.example.com` as a subdomain wildcard. Use `*` to allow all embeds (potentially insecure and conflicting with `DATA_EMBED_WHITELIST`)",
-      "default": "www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com"
+      "default": "www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com, www.slideshare.net"
```

Comment:
`www.slideshare.net` is now listed twice...

Reply:
well this is embarrassing, don't know how I completely missed that. I'll update this, I'm trying to figure out why the new embeds won't work, you can expect a PR soon that will correct this.